app/models/stripe_card.rb

```diff
@@ -303,7 +303,14 @@ def sync_from_stripe!
     self.last4 = stripe_obj[:last4]
     self.stripe_status = stripe_obj[:status]
     self.card_type = stripe_obj[:type]
-    self.stripe_card_personalization_design_id = StripeCard::PersonalizationDesign.find_by(stripe_id: stripe_obj[:personalization_design])&.id
+    # On ~2024-03-26, Stripe introduced personalization designs for physical cards
+    # This resulted in older cards not having a personalization design ID.
+    # This fix checks if its an old card without a personalization design ID and sets it to the default black design ID (311).
+    if self.created_at < Time.utc(2024, 3, 26) && stripe_obj[:personalization_design].nil?
```

Comment:
The first Stripe Card we found with a null personalization design (according to Stripe) was created at `2024-03-26 05:02:30.268839000 UTC +00:00`, which means this date likely needs to be `< 2024-3-27`